src/firebase/models/players_in_room.js

```diff
@@ -0,0 +1,34 @@
+import { ref, onValue, onDisconnect, set, child } from "firebase/database";
+import rtdb from "../rtdb";
```

Comment:
Importing the realtime database. We could have broken this out, but since it's only used here, not necessarily worth doing

Reply:
Actually after seeing rtdb.js I see that I might not have fully understood...